lib/internal/errors.js

```diff
@@ -1434,8 +1434,7 @@ E('ERR_UNSUPPORTED_DIR_IMPORT', "Directory import '%s' is not supported " +
 E('ERR_UNSUPPORTED_ESM_URL_SCHEME', (url) => {
   let msg = 'Only file and data URLs are supported by the default ESM loader';
   if (isWindows && url.protocol.length === 2) {
-    msg += '. Absolute Windows paths without prefix are not valid URLs, ' +
-      "consider using 'file://' prefix";
+    msg += ". Absolute Windows paths need a 'file://' prefix.";
```

Comment:
```suggestion
    msg += ". Absolute Windows paths need a 'file://' prefix";
```
The subsequent append to `msg` will add the period. (Or ideally the periods should be at the end of the sentences they belong to.)